docs/getting-started/getting-started-sql-snowflake.md

```diff
@@ -57,30 +56,39 @@ You can read more about Fabrics [here.](/docs/concepts/fabrics/fabrics.md)
-| **8 Database** - Specify the desired Snowflake database for default writes for this execution environment.                                                                                                                                                                                                                                                                             |
-| **9 Schema** - Specify the desired Snowflake schema for default writes for this execution environment.                                                                                                                                                                                                                                                                                 |
-| **10 Continue** to complete the Fabric creation.                                                                                                                                                                                                                                                                                                                                       |
+**(1) Provider Type** - Select SQL as the Provider type.
+
+**(2) Provider** - Click the dropdown menu for the list of supported Provider types. Select Snowflake.
+
+**(3) URL** - Add the Snowflake Account URL, which looks like this: `https://<org>-<account>.snowflakecomputing.com`
+
+**(4) Username** - Add the username that Prophecy will use to connect to the Snowflake Warehouse.
+
+**(5) Password** - Add the password that Prophecy will use to connect to the Snowflake Warehouse. These username/password credentials are encrypted for secure storage. Also, each Prophecy user will provide their own username/password credential upon login. Be sure these credentials are scoped appropriately; Prophecy respects the authorization granted to this Snowflake user.
+
+**(6) Role** - Add the Snowflake [role](https://docs.snowflake.com/en/user-guide/security-access-control-overview#roles) that Prophecy will use to read data and execute queries on the Snowflake Warehouse. The role must be already granted to the username/password provided above and should be scoped according to the permission set desired for Prophecy.
+
+**(7) Warehouse** - Specify the Snowflake warehouse for default writes for this execution environment.
+
+**(8) Database** - Specify the desired Snowflake database for default writes for this execution environment.
+
+**(9) Schema** - Specify the desired Snowflake schema for default writes for this execution environment.
+
+**(10) Continue** - to complete the Fabric creation.
 
 :::info
 Each user can read tables from each database and schema for which they have access. The default write database and schema is set here in the Fabric.
```

Comment:
Override in model configuration looks to be a forthcoming feature, so I would wait until it's ready. I want to avoid confusing the reader with dbt's `custom schema`.